stories/templates/Accordion/basic.js

```diff
@@ -2,9 +2,11 @@ import { html } from "lit-html";
 
 export const Template = args =>
   html`
-    <sgds-accordion-item summary="Solo Accordion" accordionItemClasses="mb-4">
-      Lorem ipsum dolor sit amet consectetur adipisicing elit. Maiores soluta eaque fugit fuga distinctio?
-    </sgds-accordion-item>
+    <sgds-accordion>
+      <sgds-accordion-item summary="Solo Accordion" accordionItemClasses="mb-4">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Maiores soluta eaque fugit fuga distinctio?
+      </sgds-accordion-item>
+    </sgds-accordion>
     <sgds-accordion .allowMultiple=${args.allowMultiple} .accordionClasses=${args.accordionClasses}>
```

Comment:
smaill fixes here.  both `accordionClasses` and `accordionitemClasses` showing `string` `undefined` label in storybook
![Screenshot 2023-05-18 at 5 40 11 AM](https://github.com/GovTechSG/sgds-web-component/assets/6713735/4985d7e2-f633-4674-97c0-39878a39c5d4)

Reply:
other than that is all good!! 👍